### ISSUE TITLE
WCAG 3 Draft Review: Response to feedback requested in 6.1 Conformance Levels

### ISSUE THREAD

**Issue by maryjom** [w3c/silver#607]
The question posed in Section 6.1 Conformance Levels was:

> We seek feedback on whether this (conformance model) flexibility will be beneficial in encouraging content providers to meet conformance because it is more achievable or whether content providers are less likely to improve accessibility if they aren't required to. We also seek feedback on the conformance approach as a whole.

**Answer:** A flexible conformance model will be beneficial to acknowledge the reality of quickly moving and changing technology and help teams working on large and complex websites or web-based applications focus on designing and developing for accessibility for critical workflows first. It could help them hone in on making them not just technically accessible, but more usable as well. I don’t see it as a method of simply defining workflows, making them accessible and ignoring everything else. I see it more as providing stepping stones on the path to higher levels of accessibility and being able to demonstrate that accessibility is improving over time.  
 
However, I am not sure if the current definition of Bronze as having a score of 3.5 for each functional category will be reasonable for all categories. I do not know how many new, less easily tested or clearly measurable criteria are being added in WCAG 3. If there are a lot of them in a functional category, there will be quite a lot of work to do before even a Bronze level can be achieved.

**Comment by cwadamsoforacle** [comment]
Proposed response:

This content has been updated with the release of the latest draft of WCAG 3, which can be found here:  [W3C Accessibility Guidelines (WCAG) 3.0](https://www.w3.org/TR/wcag-3.0/).  We believe that these issues are no longer relevant based on the changes made to the latest working draft.  Should you feel that these issues remain in the current draft, please log a new issue here:  [New Issue · w3c/wcag3](https://github.com/w3c/wcag3/issues/new).

**Comment by maryjom** [comment]
I agree that the current conformance content better addresses the concerns in this issue. Therefore, I'm closing it.
